public/document-table-vis.js

```diff
@@ -18,104 +18,113 @@
  */
 
 import { i18n } from '@kbn/i18n';
-import { Schemas } from '../../../src/legacy/core_plugins/vis_default_editor/public';
-import { prepareJson, prepareString } from '../../../src/legacy/core_plugins/visualizations/public/np_ready/public/legacy/build_pipeline';
+import { Schemas } from '../../../src/plugins/vis_default_editor/public';
+import { prepareJson, prepareString } from '../../../src/plugins/visualizations/public/legacy/build_pipeline';
 
 import tableVisTemplate from './enhanced-table-vis.html';
-import { EnhancedTableVisualizationController } from './vis_controller';
+import { getEnhancedTableVisualizationController } from './vis_controller';
 import { enhancedTableRequestHandler } from './data_load/enhanced-table-request-handler';
 import { documentTableResponseHandler } from './data_load/document-table-response-handler';
 import { DocumentTableData } from './components/document_table_vis_data';
 import { EnhancedTableOptions } from './components/enhanced_table_vis_options';
 
+const toExpression = (vis) => {
+  const visConfig = { ...vis.params };
+  const { indexPattern, aggs } = vis.data;
+  let pipeline = `kibana | enhanced_table_visualization type='${vis.type.name}'
+    ${prepareJson('visConfig', visConfig)}
+    ${prepareJson('aggConfigs', aggs.aggs)}
+    metricsAtAllLevels=${vis.isHierarchical()}
+    partialRows=${vis.type.requiresPartialRows || vis.params.showPartialRows || false} `;
+  if (indexPattern) {
+    pipeline += `${prepareString('index', indexPattern.id)}`;
+  }
+  return pipeline;
+};
```

Comment:
could you remove trailing `;`?

this is because of my previous change request :)